src/plugins/intel_cpu/src/mkldnn_plugin.cpp

```diff
@@ -172,7 +172,7 @@ static void TransformationUpToCPUSpecificOpSet(const std::shared_ptr<ngraph::Fun
     manager.register_pass<ngraph::pass::InitNodeInfo>();
 
     const bool useLpt =
-            _enableLPT &&
```

Comment:
This alignment fix is absolute correct, but other updates are not connected LPT and can affect other team members. Can you, please, rollback alignment changes which is not connected with LPT? thanks

Reply:
done, thank you.